src/CommunityToolkit.Maui.Core/Primitives/Defaults/AvatarViewDefaults.shared.cs

```diff
@@ -0,0 +1,36 @@
+﻿namespace CommunityToolkit.Maui.Core;
+
+using System.ComponentModel;
+
+/// <summary>Default Values for <see cref="IAvatarView"/></summary>
+[EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
We don't like to have secrets on `Core`. So not sure if we want to hide this.

Reply:
> We don't like to have secrets on `Core`. So not sure if we want to hide this.

@pictos I was using `AlertDefaults` and `DrawingViewDefaults` as my guide.  These have the same _EditorBrowsable_ decorations.  I'll change to whatever is best?

Reply:
Yup, George's implementation is totally valid and we should keep the `EditorBrowsable` attribute. 

Like `AlertDefaults`, these are internal defaults for our custom controls.